src/ext/java/org/opentripplanner/ext/legacygraphqlapi/model/LegacyGraphQLTicketType.java

```diff
@@ -0,0 +1,29 @@
+package org.opentripplanner.ext.legacygraphqlapi.model;
+
+import java.util.Set;
+import org.opentripplanner.routing.core.FareRuleSet;
+
+public class LegacyGraphQLTicketType {
```

Comment:
Do we need this, can't we just make `LegacyGraphQLTicketTypeImpl´ operate on `FareRuleSet`

Reply:
Yes, we can actually use the `FareRuleSet` as it has all the necessary information